core/src/test/java/com/sequenceiq/cloudbreak/converter/v2/StackV4RequestToTemplatePreparationObjectConverterTest.java

```diff
@@ -438,6 +440,18 @@ public void testStackPlacement() {
         assertEquals(REGION, result.getPlacementView().get().getAvailabilityZone());
     }
 
+    @Test
+    public void testHostGroupViewWithoutVolumeCount(){
+        List<InstanceGroupV4Request> instanceGroups = createInstanceGroups();
+        HostgroupView test=  new HostgroupView(
+                instanceGroups.get(0).getName(),
+                instanceGroups.get(0).getTemplate().getAttachedVolumes().stream().mapToInt(VolumeV4Request::getCount).sum(),
+                instanceGroups.get(0).getType(),
+                instanceGroups.get(0).getNodeCount());
+
+         assertNotNull(test);
+    }
+
```

Comment:
Instead of adding the test here please create a new test file, `TemplateToInstanceTemplateV4RequestConverterTest`, in `com.sequenceiq.cloudbreak.converter.v2` package, similarly to the other converter tests in that package and add the unit test there for the change.